web/src/utils/constants.ts

```diff
@@ -56,15 +56,15 @@ export const modeColor: { [mode in ElectricityModeType]: string } = {
 };
 
 export const modeOrder = [
```

Comment:
Adding a comment to explain the logic here so it's clear :)
```suggestion
/**
 * The order of modes use the following logic:
 * 1. renewables first (sorted alphabetically)
 * 2. low carbon (nuclear)
 * 3. fossil fuels (sorted alphabetically)
 * 4. storage (sorted alphabetically)
 */
```